Modules/mmapmodule.c

```diff
@@ -1951,6 +1952,7 @@ new_mmap_object(PyTypeObject *type, PyObject *args, PyObject *kwdict)
         PyErr_SetFromErrno(PyExc_OSError);
         return NULL;
     }
+    _PyAnnotateMemoryMap(m_obj->data, map_size, "cpython:mmap:new_mmap_object");
```

Comment:
I don't think that it's useful to mention the internal function name:

```suggestion
    _PyAnnotateMemoryMap(m_obj->data, map_size, "cpython:mmap");
```